src/main/java/net/mcreator/ui/init/L10N.java

```diff
@@ -182,6 +183,10 @@ public static JButton button(String key, Object... parameter) {
 		return new JButton(t(key, parameter));
 	}
 
+	public static TechnicalButton funcbutton(String key, Object... parameter) {

```

Comment:
Call it technicalbutton, if it returns technical button

Reply:
`funcbutton` was way shorter...

Reply:
✔️ Fixed